week2/index.js

```diff
@@ -0,0 +1,135 @@
+
+const fs = require('fs');
+const readline = require('readline');
+let todos = [];
+
+
+if (process.argv[2] !== undefined) {
+    start();
+} else {
+    console.log('Please specify an option to use, here is the manual\n');
+    manualReader('./help.txt');
+}
+
+
+function start() {
+    switch (process.argv[2].trim().toLowerCase()) {
+        case 'help':
+            manualReader('./help.txt');
+            break;
+        case 'add':
+            checkFileContent().then((item) => addTodoItem(item)).catch((err) => console.log(err));
+            break;
+        case 'remove':
+            itemRemover(process.argv[3]);
+            break;
+        case 'list':
+            console.log('The items on your todo list: ');
+            todoFileReader('./todos.txt');
+            break;
+        case 'reset':
+            itemsResetter();
+            break;
+        default:
+            console.log('No such option! Take a look at the manual\n');
+            manualReader('./help.txt');
+            break;
+    }
+}
+
+
+function manualReader(fileName) {
```

Comment:
You could extract your two reader functions a bit better by having one reader function and just calling a callback with the data or returning a promise that resolves with the data, rather than having two functions doing similar things. This way you can let the 'caller' decide what to do with the resulting data, not have the code that comes after reading the file inside these functions.

This better follows the pattern of 'utility functions', where your functions have one key job and they do only that one thing.